examples/tick-ami/tick.json

```diff
@@ -106,6 +107,10 @@
     "type": "file",
     "source": "{{template_dir}}/../influxdb-ami/config/influxdb.conf",
     "destination": "/tmp/config/influxdb.conf"
+  },{
+    "type": "file",
+    "source": "{{template_dir}}/../chronograf-ami/config/chronograf",
```

Comment:
is chronograph config not a `.conf` file ?

Reply:
Nope, it's a file with no extension. See here: https://docs.influxdata.com/chronograf/v1.7/administration/configuration/#setting-custom-default-chronograf-config-options